static/scripts/video-player/webcomponents-lite.js

```diff
@@ -0,0 +1,221 @@
+/**
```

Comment:
use https://github.com/schul-cloud/schulcloud-client/blob/master/static/vendor/webcomponents-lite.min.js instead

Reply:
Ich mach das

Reply:
Wir verwenden jetzt das aktuelle https://github.com/webcomponents/webcomponentsjs/blob/master/webcomponents-lite.js im video player und in der homework\overview.hbs, funktioniert.